imap_processing/hi/l1c/hi_l1c.py

```diff
@@ -276,8 +290,8 @@ def pset_geometry(pset_et: float, sensor_str: str) -> dict[str, xr.DataArray]:
     el = 0 if "90" in sensor_str else -45
     dps_az_el = np.array(
         [
-            np.arange(0.05, 360, 0.1),
-            np.full(3600, el),
+            SPIN_PHASE_BIN_CENTERS * 360,
```

Comment:
Why do we need to multiply by 360 here? Above your SPIN_PHASE_BIN_CENTERS are already in degrees I believe.
```suggestion
            SPIN_PHASE_BIN_CENTERS,
```

Reply:
No, from above: `SPIN_PHASE_BIN_EDGES = np.linspace(0, 360, N_SPIN_BINS + 1) / 360`. Maybe it just confuses things to not just have `SPIN_PHASE_BIN_EDGES = np.linspace(0, 1, N_SPIN_BINS + 1)`?

This is something I will add to the agenda for my next Hi tag-up. Paul uses the term `spin-phase` to mean fraction of a spin. So, `spin_phase = spin_angle_degrees / 360`. That is what he has asked for throughout the algorithm document for L1. I don't know if Hi is the only instrument using spin-phase or not. Seems like yet another area to get consistency across instruments.

Reply:
OK, this makes sense to me, and I just didn't realize which one was in 0-1 and which one was 0-360. Not reading close enough on my part!

> Seems like yet another area to get consistency across instruments.